homeassistant/components/lupusec/entity.py

```diff
@@ -0,0 +1,48 @@
+"""Provides the Lupusec entity for Home Assistant."""
+from homeassistant.helpers.entity import Entity
+
+from .const import DOMAIN
+
+
+class LupusecDevice(Entity):
+    """Representation of a Lupusec device."""
+
+    _attr_has_entity_name = True
+
+    def __init__(self, data, device, config_entry) -> None:
```

Comment:
Please pass in only the entry_id instead of the full config_entry

Reply:
Done.